core/state/state_object.go

```diff
@@ -478,7 +514,6 @@ func (s *StateObject) deepCopy(db *StateDB) *StateObject {
 	}
 	stateObject.code = s.code
 	stateObject.dirtyStorage = s.dirtyStorage.Copy()
-	stateObject.originStorage = s.originStorage.Copy()
```

Comment:
Why delete this?

Reply:
I deleted it before using Layer 1.5 ,it should not deleted when using Layer 1.5